### DISCUSSION TITLE
🙍‍♂️ No further development in sight at the moment?

### DISCUSSION THREAD

**Discussion by musiolmarco** [flutterdata/flutter_data#285] [General]
Hey I don't want to put any pressure on the author(s) and I love the idea of the package, but could it be that development has just stopped? 

I was really looking forward to 2.0, but unfortunately I haven't heard anything about the package for a long time.🙍‍♂️

**Comment by frank06** [comment]
Unfortunately I had some personal issues and had to step away for a few months. Sorry for the lack of updates. I am still unsure about my future involvement but there will be development on v2, resuming around November for sure. What I don't know is how much I will keep contributing. I am talking to a long-time FD user, and good Dart developer who will either be helping me or continuing on v2.

**Reply by musiolmarco** [reply]
All right, no problem. I hope nothing bad has happened.

Thanks for your reply🙏

**Comment by frank06** [comment]
I just released 2.0.0, I heard from clients that it has been working great for their apps, in production . I am not too involved anymore but I can keep some maintenance.
